sympy/sets/sets.py

```diff
@@ -1959,3 +1957,32 @@ def simplify_intersection(args):
         return args.pop()
     else:
         return Intersection(args, evaluate=False)
+
+def set_add(x, y):
+    from sympy.sets.handlers.add import _set_add
+    from sympy.sets.setexpr import _apply_operation
```

Comment:
`_apply_operation` and `_handle_finite_sets` should also be moved, in my opinion.

Reply:
do I need to move it? @lilithxxx